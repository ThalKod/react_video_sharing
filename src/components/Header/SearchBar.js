import React from "react";

export default class SearchBar extends React.Component {
  render(){
    return(
        <div className="topsearch">
          <div className="input-group">
            <span className="input-group-addon" id="sizing-addon2"><i className="fa fa-search"></i></span>
            <input type="text" className="form-control" placeholder="Search" aria-describedby="sizing-addon2"/>
          </div>
        </div>
    )
  }
}
