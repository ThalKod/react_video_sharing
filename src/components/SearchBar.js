import React from "react";

export default class SearchBar extends React.Component {
  render(){
    return(
        <div>

        </div>
    )
  }
}
