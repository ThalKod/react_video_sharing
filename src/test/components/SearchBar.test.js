import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

import SearchBar from "../../components/Header/SearchBar";


describe("SearchBar", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<SearchBar/>)
  });


  it("should render the SearchBar correctly", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  })
});
