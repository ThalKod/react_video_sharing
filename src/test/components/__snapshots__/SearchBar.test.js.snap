// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar should render the SearchBar correctly 1`] = `
<div
  className="topsearch"
>
  <div
    className="input-group"
  >
    <span
      className="input-group-addon"
      id="sizing-addon2"
    >
      <i
        className="fa fa-search"
      />
    </span>
    <form
      onSubmit={[Function]}
    >
      <input
        aria-describedby="sizing-addon2"
        className="form-control"
        onChange={[Function]}
        placeholder="Search"
        type="text"
        value=""
      />
    </form>
  </div>
</div>
`;
